base/system/diskpart/diskpart_msg.mc

```diff
@@ -7219,84 +7237,87 @@ Example:
     SET ID=ebd0a0a2-b9e5-4433-87c0-68b6b72699c7
 .
 Language=Romanian
-    Changes the partition type field for the partition with focus.
+    Schimbă registru tipului partiției pentru partiția în cauză.
 
-Syntax:  SET ID={<BYTE> | <GUID>} [OVERRIDE] [NOERR]
+Sintaxă:  SET ID={<BYTE> | <GUID>} [OVERRIDE] [NOERR]
 
     ID={<BYTE> | <GUID>}
 
-                Specifies the new partition type.
-                For master boot record (MBR) disks, you can specify a partition
-                type byte, in hexadecimal form, for the partition. Any
-                partition type byte can be specified with this parameter except
-                for type 0x42 (LDM partition). Note that the leading '0x' is 
-                omitted when specifying the hexadecimal partition type.
-
-                For GUID partition table (GPT) disks you can specify a
-                partition type GUID for the partition. Recognized GUIDs
-                include:
-
-                    EFI System partition:
+                Specifică noul tip al partiției.
+                Pentru discurile cu Registrul principal de inițializare (MBR),
+                puteți speciifica o partiție de tip octet, în formă
+                hexazecimală, pentru partiție. Orice partiție de tip octet
+                poate fi specificată cu acest parametru, exceptând pentru tipul
```

Comment:
[My previous comment on translation for "partition type byte".](https://github.com/reactos/reactos/pull/5485#discussion_r1285223016)
The mentioned byte is not part of partition type, but is actually specifying the type of partition, so a more accurate translation (in the first sentence) may be "puteți specifica un octet, în format hexazecimal, pentru un tip de partiție". (A similar formula goes for the second sentence too.)